JSDemos/Demos/DataGrid/GridColumnsHidingPriority/description.md

```diff
@@ -1 +1,3 @@
-The **DataGrid** can hide columns according to&nbsp;the specified priority if&nbsp;screen width is&nbsp;not enough to&nbsp;display all of&nbsp;them. The hiding priority is&nbsp;specified using the **hidingPriority** field of&nbsp;an&nbsp;item in&nbsp;the **columns** array. Switch between the landscape and portrait layouts to&nbsp;see how the **DataGrid** looks on&nbsp;wide and narrow screens.
\ No newline at end of file
+When the **DataGrid** adapts its layout to small screen width, it hides columns based on [hidingPriority](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/columns/#hidingPriority). Columns with lower values are hidden first. 
+
+In this demo, the `CustomerStoreCity` column is hidden in both vertical and horizontal orientation modes because its **hidingPriority** is 0. The `CustomerStoreState` and `OrderDate` columns have priorities 1 and 2. The **DataGrid** hides these columns after the `CustomerStoreCity` column, only when the screen orientation is vertical.
```

Comment:
```suggestion
In this demo, the `CustomerStoreCity` column is hidden in vertical and horizontal orientation modes because its **hidingPriority** is 0. The `CustomerStoreState` and `OrderDate` columns have a priority of 1 and 2. The **DataGrid** hides these columns after the `CustomerStoreCity` column, only when the screen orientation is vertical.
```